Observer/Xamarin.Forms.HotReload.Observer/FileObserver.cs

```diff
@@ -88,6 +92,12 @@ private static string RetrieveCommandLineArgument(string key, string defaultValu
         
         private static void OnFileChanged(object source, FileSystemEventArgs e)
         {
+            string fullpath = e.FullPath;
+            if (Regex.IsMatch(fullpath, "(/|^)((obj)|(bin))/"))
```

Comment:
Do you think it is important check?
Somebody will face with issue, if he/she has solution directory with such name (obj/bin). Yes, i understand, that is is very rare case, but in theory..

Reply:
I agree with you. I need your feedback before doing some testing with the option to ignore the obj/bin folder for removing ignore `bin/obj` logic. Did you question based on understanding of the `obj / bin` folder? To me, that space is an unknown world. If you have an understanding of the space or have good learning resources, share please.

The reason that I ignored `obj / bin` is I was worried that there would be some unknown collision because the space is outside my control. I was also worried that how many network loads might be large because I do not know how often the file changes in that area.

Though I thought the problem you mentioned for a while when I code, I did this because I thought it was less important than the problem I were worried about.

I've written unsophisticated code. This part is what I should have left with the PR request, but I missed it. Thank you for pointing out.

Reply:
@wplong11 hm, seems xaml files aren't copied to obj / bin folders

I think, we should remove this line